src/Analyser/NodeScopeResolver.php

```diff
@@ -534,6 +534,7 @@ private function processStmtNode(
 						false,
 						$scope->isInTrait(),
 						$scope->getClassReflection()->isReadOnly(),
+						false
```

Comment:
It seems that there is no processing to get the PHPDoc of the parameter of the constructor promotion, so I ignored it with `false`.